buildFont.sh

```diff
@@ -25,11 +25,28 @@ tx -cff2 +S +b -std "$otf_file" "$folder/.tb_cff2" 2> /dev/null
 # replace CFF2 table with subroutinized version
 sfntedit -a CFF2="$folder/.tb_cff2" "$otf_file" 1> /dev/null
 
+# TEMP HACK until fontTools.feaLib supports STAT
 # comment out STAT feature file which cannot be digested by fontmake
 sed -i '' 's/^/#/' $folder/STAT.fea
 
-# build the TTF version
+# set +e so we have the opportunity to un-hack STAT.fea before exiting if
+# there's a problem with fontmake.
+set +  # 
```

Comment:
Is there a letter missing here?

Reply:
Indeed! Fixed in https://github.com/adobe-fonts/adobe-variable-font-prototype/pull/14/commits/1ef715e49a253523349a65922da985e97354a909